packages/edit-site/src/components/left-sidebar/navigation-panel/template-navigation-items.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	Button,
+	__experimentalNavigationItem as NavigationItem,
+} from '@wordpress/components';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import TemplatePreview from './template-preview';
+import { NavigationPanelPreviewFill } from '.';
+
+const TITLES = {
```

Comment:
What do you think about moving this constant into its own file?

Reply:
Actually, maybe even all constants in this PR could live in a `constants.js` file.
`GENERAL_TEMPLATES`, `GENERAL_TEMPLATE_SLUGS`, and `TITLES` here.
It should make it easier to look up and to update. 🤔

Reply:
Great idea! Done!